docs/source/architecture/service_fees.rst

```diff
@@ -3,89 +3,67 @@
+The minimum fee rate applies to each individual worker machine managing a given sharing policy. In other words, the rate is also *per Ursula*. If Alice wishes to employ multiple Ursulas to service a single sharing policy (``n`` > 1), a common configuration, then they must pay each staker associated with that policy the same fee rate. This rate will be the highest from the set of *minimum* fee rates specified by the stakers with which they engage. Alices may attempt price optimization strategies to find the cheapest group of Ursulas.
 
-The minimum sum of fees a staker can receive period-to-period is the product of their specified minimum fee rate and the number of active sharing policies their Ursula has been assigned. 
+When issuing a sharing policy, Alices are required to escrow a deposit covering the cost of the entire duration of the policy. This deposit is split and paid out in tranches to stakers once per period. The precise payment flow follows a repeated three-period cycle; in the first period the Ursula *makes a commitment* to the second period. Then, the Ursula services the policy during the second period (and makes a commitment to the third period). In the third period, they receive the fee tranche for their work during the second period, and continue servicing the policy, etc. This cycle continues until all the policies that Ursula is responsible for have expired. Note that *making a commitment* was formerly referred to as *confirming activity*.
 
-Note that *making a commitment* was formerly referred to as *confirming activity*.
+The minimum sum of fees a staker can receive period-to-period is the product of their specified minimum fee rate and the number of active sharing policies their Ursula has been assigned.
 
 
 Global fee range
 ----------------
 
-The global fee range is a means of establishing quasi-universal pricing for the NuCypher service. It is enforced via the function ``feeRateRange`` in ``PolicyManager.sol``, which specifies per sharing policy and per 24h period constraints expressed in **WEI**. In other material, rates are discussed in **GWEI** (and fiat).
+The global fee range is a means of establishing quasi-universal pricing for the NuCypher service. It is enforced via the function ``feeRateRange`` in ``PolicyManager.sol``, which specifies per sharing policy and per 24h period constraints expressed in **WEI**. Note that elsewhere, fee rates are discussed in **GWEI** and fiat (USD).
 
 The minimum fee rate chosen by stakers must fall within the global fee range. The network will launch with the following parameters:
 
 **Minimum fee rate**
 
-XX GWEI (~$X.XX x10^-X) *per period*
-
-XX,XXX GWEI ($X.XXX) *per year*
+50 GWEI (~$1.59 x10^-5) *per period*
```

Comment:
As gas prices change, so will this number - it might be better to display only the *gwei* denomination here, and not USD.

Reply:
I do think they provide a useful reference (with the static exchange rate stated below as a caveat), but happy to remove. I remember @derekpierre had a similar view. Although these conversions change with the ETHUSD rate, not with fluctuations to viable gas prices – so not nearly as volatile!